src/client/tile.h

```diff
@@ -243,6 +244,10 @@ struct TileLayer
 	void applyMaterialOptions(video::SMaterial &material) const
 	{
 		switch (material_type) {
+		case TILE_MATERIAL_SOLID:
```

Comment:
Well I was thinking `TILE_MATERIAL_SOLID` more reflected the actual material type `EMT_SOLID` but I can change that if necessary.